src/bb-config-sample.php

```diff
@@ -174,7 +174,7 @@
          * @since 4.22.0
          * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
          */
-        'user_agent' => 'Mozilla/5.0 (RedHatEnterpriseLinux; Linux x86_64; BoxBilling; +http://boxbilling.org) Gecko/20100101 Firefox/93.0',
+        'user_agent' => 'Mozilla/5.0 (RedHatEnterpriseLinux; Linux x86_64; FOSSBilling; +http://boxbilling.org) Gecko/20100101 Firefox/93.0',
```

Comment:
sam with the url here, we have the https://fossbilling.org

Reply:
Done